src/vtbackend/Screen.cpp

```diff
@@ -1852,7 +1852,8 @@ void Screen<Cell>::renderImage(shared_ptr<Image const> image,
     // TODO: make use of imageOffset
     (void) imageOffset;
 
-    auto const linesAvailable = _settings.pageSize.lines - topLeft.line.as<LineCount>();
+    auto const linesAvailable = _settings.pageSize.lines - topLeft.line.as<LineCount>()
+                                - (_terminal.state().statusDisplayType == StatusDisplayType::None ? 0 : 1);
```

Comment:
I think this should be a little bit more future-proof, because the line height of the statusline does not necessarily have to be 1 all the time.

You'd need to fetch the value from `_indicatorStatusScreen` and `_hostWritableStatusLineScreen`'s `pageSize().lines`. They do not necessarily need to match either.

Currently that's `1` yes, but it may and most likely will change in the future. Imagine using the statusline to also display multiline error messages (not done yet, but likely in the future), that would be a value `> 1` then :).